aws/resource_aws_dms_replication_task_test.go

```diff
@@ -104,12 +109,48 @@ func dmsReplicationTaskDestroy(s *terraform.State) error {
 
 func dmsReplicationTaskConfig(randId string) string {
 	return fmt.Sprintf(`
+data "aws_iam_policy_document" "dms_assume_role_policy_document" {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      identifiers = ["dms.amazonaws.com"]
+      type        = "Service"
+    }
+  }
+}
+
+resource "aws_iam_role" "dms_vpc_role" {
+  assume_role_policy = data.aws_iam_policy_document.dms_assume_role_policy_document.json
+  name = "dms-vpc-role"
+}
+
+resource "aws_iam_role" "dms_cloudwatch_logs_role" {
+  assume_role_policy = data.aws_iam_policy_document.dms_assume_role_policy_document.json
+  name = "dms-cloudwatch-logs-role"
+}
+
+resource "aws_iam_role_policy_attachment" "dms_vpc_access_policy" {
+  role       = aws_iam_role.dms_vpc_role.name
+  policy_arn = "arn:aws:iam::aws:policy/service-role/AmazonDMSVPCManagementRole"
+}
+
+resource "aws_iam_role_policy_attachment" "dms_cloudwatch_logs_access_policy" {
+  role       = aws_iam_role.dms_cloudwatch_logs_role.name
+  policy_arn = "arn:aws:iam::aws:policy/service-role/AmazonDMSCloudWatchLogsRole"
+}
```

Comment:
[Configuring DMS via the API requires two access policies](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Security.html#CHAP_Security.APIRole), without them you'll get an error:

`Error: AccessDeniedFault: The IAM Role arn:aws:iam::<accountid>:role/dms-vpc-role is not configured properly.`

I'm assuming that people run these tests against an empty AWS account, but if that's not the case, then this may cause failures because the role names are fixed.